### ISSUE TITLE
StructuralProfiles Question

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ebouhoubeiny/cpacs#221]
```
Occurs in xPath: 
/cpacs/vehicles/profiles/structuralprofiles

Problem/Question: 
StructuralProfiles Question

Comments: 
Why is the structural profile definition based on actual point coordinate 
values and not based on a 'unit-profile' as is the case for wingAirfoils and 
fuselageProfiles? Setting the actual dimension can then be done using a single 
UID+parameter combination (after which materials and thicknesses need to be 
assigned to sheets). Maybe it is necessary to assign a choice for 
globalBeamProperties and pointand sheetProperties

Example: 
Scaling should be possible ?!

Reply: 
Needs further discussion CPACS 2.1. Include Dieter in discussion

daniel.boehnke@dlr.de
```

Original issue reported on code.google.com by `daniel.b...@googlemail.com` on 25 Jan 2012 at 10:56
- Merged into: #339

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 8 Feb 2012 at 9:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 9 Feb 2012 at 3:10
- Added labels: **milestone-CPACS2.1**

**Comment by GoogleCodeExporter** [comment]
```
This is partly solved by new profile types in CPACS 2.01. (and subsequently 
2.1) 

There will be a definition for typical items like C, I, T, Z stringers and 
these can be scaled according to dimensioning. 

Should discuss this point again...
```

Original comment by `daniel.b...@googlemail.com` on 7 Oct 2012 at 11:38
- Added labels: **milestone-CPACS2.01**
- Removed labels: **milestone-CPACS2.1**

**Comment by GoogleCodeExporter** [comment]
Original comment by `daniel.b...@googlemail.com` on 30 Oct 2012 at 2:34
- Changed state: **Duplicate**
